website/common/script/content/bundles.js

```diff
@@ -12,7 +12,7 @@ const bundles = {
   featheredFriends: {
     key: 'featheredFriends',
     text: t('featheredFriendsText'),
-    notes: t('featheredFriendsNotes'),
+    notes: t('featheredFriendsNotes', { date: moment('2023-09-30').format('LL') }),
```

Comment:
This is on the right track! I think we can go one step better, though: instead of continuing to hardcode the date here, use the end date of the associated event.